tests/test_data/settings_old

```diff
@@ -92,7 +92,7 @@ cheetah_import_whitelist:
 createrepo_flags: "-c cache -s sha"
 
 # if no autoinstall template is specified to profile add, use this template
-default_autoinstall: /var/lib/cobbler/autoinstall_templates/default.ks
+default_autoinstall: /var/lib/cobbler/templates/default.ks
```

Comment:
@SchoolGuy Are you sure? It could still get adjusted, it was using the old "/var/lib/cobbler/autoinstall_templates" as well. So it should not hurt and looks cleaner to me?

Reply:
Yes I am. This is for for migration from an old settings file to a new one. This should never be adjusted to be able to simulate an old file, updating this could possibly cause incorrect behavior of the upgrade tests.